From e3dd48d53f8241e331d5c80153b44830ef1e62ed Mon Sep 17 00:00:00 2001
From: Johannes Thoma <johannes@johannesthoma.com>
Date: Tue, 2 Apr 2019 19:51:31 +0200
Subject: [PATCH 1/3] drbd_req: typeinfo for a idr_foreach loop.

Since MS C does not have typeof, we need to explicitly state
types for loop macros.
---
 drbd/drbd_req.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drbd/drbd_req.c b/drbd/drbd_req.c
index 13462f9..90dd959 100644
--- a/drbd/drbd_req.c
+++ b/drbd/drbd_req.c
@@ -757,7 +757,7 @@ static void mod_rq_state(struct drbd_request *req, struct bio_and_error *m,
 			 * go Ahead -> SyncSource tries to trigger that
 			 * transition for *all* peer devices currently in
 			 * L_AHEAD for this connection. */
-			idr_for_each_entry(struct  *, &peer_device->connection->peer_devices, pd, vnr) {
+			idr_for_each_entry(struct drbd_peer_device*, &peer_device->connection->peer_devices, pd, vnr) {
 				if (pd->repl_state[NOW] != L_AHEAD)
 					continue;
 				if (test_and_set_bit(AHEAD_TO_SYNC_SOURCE, &pd->flags))
-- 
2.17.0

